### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by iLynette**
In this project we implemented the following;

-  Create a collection that keeps a list of books.

- Create a function to add a new book to the collection, with title and author.

- Create a function to remove a book from the collection.

- Display all books saved in the collection in the top part of the page.

- When a user clicks the "Add" button, a new book is added to the collection. The new book is displayed on the page.

- When a user clicks the "Remove" button, the correct book is removed from the collection. The correct book disappears from the page.

- Preserve the data in the browser's memory by using local storage

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    });
+    display();
+    addLocalStorage();
+  }
+}
+function removeBook(id) {
+  books = books.filter((e) => e.id != id);
+  display();
+  addLocalStorage();
+}
```
Comment (mikethreels) [inline script.js]: - [ ] when you have a list of books and the user removes a book and after that, the user adds another book there can be 2 books with the same id, if the user tries to remove either of these 2 books after that both books will disappear. you can try to fix this by updating the id of all books after removing a book.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Tufahel) [reply script.js]: Thank you for your review. We have worked on that and solved it.

**Review by rustam-tolipov — APPROVED** [review]
## Hi team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**